guided-match/dml-versions/v024.sql

```diff
@@ -0,0 +1,202 @@
+/*
+Version No   Version Description
+----------   -------------------
+v0.0.24       Adding keywords for Furniture and Associated Services
+*/
+
+-- CREATING THE JOURNEY
+INSERT INTO journeys (journey_id,journey_name,published)
+	VALUES ('085df7f9-8014-4350-b3a4-b1eadf5c3766'::uuid,'Furniture and Associated Services',true);
+-----------------------
+
+-- CREATING FRESH TABLE
+drop table if exists temp_domain_values;
+create table temp_domain_values (search_term varchar(75),
+				 domain_name varchar(50),
+				 domain_desc varchar(2000));
+
+drop table if exists temp_journeys;
+create table temp_journeys (domain_name varchar(50),
+						    journey_uuid uuid);
+-----------------------
+
+-- INSERTING NEW KEYWORDS
+INSERT INTO temp_journeys values ('Furniture and Associated Services','085df7f9-8014-4350-b3a4-b1eadf5c3766');
+insert into temp_domain_values values
+('Accoustic Divider','Furniture and Associated Services', 'Buy a range of furniture for use in public sector organisations, as well as repair and renovation services.'),
```

Comment:
remove the full stop at the end of the sentencae

Reply:
done!